client/src/stylesheets/questions.scss

```diff
@@ -0,0 +1,304 @@
+			box-shadow: 4px 4px 4px -2px rgba(20,50,40,.5);
+		}
+		
+	}
+
+	input {
+		outline: none;
+	}
+	input[type=search]{
+		background: #ededed url(https://static.tumblr.com/ftv85bp/MIXmud4tx/search-icon.png) no-repeat 9px center;
+		border: solid 1px #ccc;
+		padding: 9px 10px 9px 32px;
+		width: 0px;
+		border-radius: 15px;
+		transition:  0.2s ease-out;
+	}
+	input[type=search]:focus{
+		width: 350px;
+		background-color: #fff;
+		border-color: #66CC75;
+		box-shadow: 0 0 5px rgba(109,207,246,.5);
+		transition:  ease-out 0.2s; 
+
+	}
+	input[type=search]:hover{
+		background-color: #fff;
+	}
+}
+
+.fixed-nav{
+	input[type=search]{
+		width: 0px;
+		border-radius: 15px;
+	}
+}
+.condense-nav{
+	.nav-link{
+		font-size: 0px;
+		transition: ease-out 0.2s;
+	}
+	.navbar-logo{
+		margin-right: 0.5em;
+	}
+}
+
+
+nav {
+  background-color: #fff;
+  top:0;
+  width: 100%;
+  transition:all 0.5s;
+  position: relative;
+  z-index: 1;
+}
+
+body.fixed-nav .container {
+  transform: scale(1);
+}
+
+body.fixed-nav nav {
+  position: fixed;
+  box-shadow:0 5px 0 rgba(0,0,0,0.1);
+}
+
+li.hidden-nav {
+  opacity: 0;
+  overflow: hidden;
+  max-width:0;
+  padding: 0;
+  background: white;
+  transition: ease-out 0.4s;
+}
+
+li.hide-nav {
+  opacity: 1;
+  max-width:500px;
+  transition: ease-out 0.4s;
+}
+
+.fixed-nav li.hide-nav  {
+  opacity: 0;
+  overflow: hidden;
+  max-width:0;
+  padding: 0;
+  background: white;
+  transition: ease-out 0.2s;
+  font-size: 1px
+
+}
+
+.nav-link{
+	color: green;
+	transition: ease-out 0.4s;
+	&:hover{
+		color: rgb(15, 79, 26);
+	}
+}
+.navbar, .ask-button, input[type=search] {
+	font-size: 12px !important;
+}
+
+
+@media (min-width: 1096px){
+	.navbar, .ask-button, input[type=search]{
+		font-size: 18px !important;
+	}
+
+	@media (min-width: 900px){
+	  .fixed-nav li.hidden-nav  {
+	    opacity: 1;
+	    max-width:500px;
+	  }
+	}
+	input[type=search]{
+		width: 100px !important;
+		border-radius: 2px !important;
+	}
+	input[type=search]:focus{
+		width: 550px  !important;
+	}
+}
+
+
+
+/******************* Questions Index Banner **********************************/
+
+.banner{
+	background: url("../images/carrots.jpg")no-repeat center center fixed;
+  background-size: cover;
+
+  .overlay{
+  	padding-bottom: 1em;
+  	padding-left: 2em;
+		padding-top: $banner-top-padding-sm;
+		background-color: rgba(0,0,0,0.3);
+  }
+
+  .page-title{
+  	display: -webkit-flex;
+  	display: flex;
+  	-webkit-flex-direction: row; /* works with row or column */
+  	flex-direction: row;
+  	-webkit-align-items: center;
+  	align-items: center;
+  	-webkit-justify-content: center;
+  	justify-content: center;
+  }
+
+  .banner-image{
+  	flex: 1;
+  	width: 60px;
+  	height: 60px;
+  }
+
+  .banner-text{
+  		flex: 8;
+  		font-family: 'Muli', sans-serif;
+  	  font-weight: 300;
+  		color: white;
+  	  text-shadow: 1.5px 1.5px 0 black; 
+  	  margin-bottom: 0px;
+  	  font-size: $banner-text-sm;
+  }
+
+  @media(min-width: 768px){
```

Comment:
Oh yeah, I've been trying on multiple screen sizes and been using chrome developer tools to make sure break points make sense and cells phones would look okay. Still some work to be done there, but the groundwork is good.